### PR TITLE
Save4

### PR DESCRIPTION
**Pull request by csmccue**
good job

### CODE REVIEW

File: render-utils.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
 export function renderPoll(question, optionA, optionB, votesA, votesB) {
     const div = document.createElement('div');
-    
+    const optionAElDiv = document.createElement('p');
+    const optionBElDiv = document.createElement('p');
     let optionQuestion = question;
```
Comment (viccaruso) [inline render-utils.js:5]: Since this doesn't get mutated it would be better to declare this as `const`

File: app.js [hunk 2]
```diff
@@ -36,57 +34,34 @@ startPollButton.addEventListener('click', () => {
...
-    votesA = 0;
-    votesB = 0;
 
-    refreshCurrentPollEl();
-    displayAllPolls();
 });
 
 increaseOptionAButton.addEventListener('click', () => {
     votesA++;
     console.log(votesA);
```
Comment (viccaruso) [inline app.js:48]: In general it is a good practice to remove any console.logs() from your code before pushing

Comment (csmccue) [reply app.js:48]: ah missed that, thanks!

File: render-utils.js (hunk 1 above)
Comment (csmccue) [reply render-utils.js:5]: roger that

**Pull request merged**